### ISSUE TITLE
Not working with latest passport version

### ISSUE THREAD

**Issue by rolodato** [bergie/passport-saml#117]
Always getting this error after a successful authentication:

```
TypeError: Cannot set property 'user' of undefined
   at /Users/rolodato/git/saml-sp-linking-sample/node_modules/passport-saml/node_modules/passport/lib/http/request.js:50:35
   at pass (/Users/rolodato/git/saml-sp-linking-sample/node_modules/passport/lib/authenticator.js:267:43)
   at serialized (/Users/rolodato/git/saml-sp-linking-sample/node_modules/passport/lib/authenticator.js:276:7)
   at /Users/rolodato/git/saml-sp-linking-sample/index.js:31:3
   at pass (/Users/rolodato/git/saml-sp-linking-sample/node_modules/passport/lib/authenticator.js:284:9)
   at Authenticator.serializeUser (/Users/rolodato/git/saml-sp-linking-sample/node_modules/passport/lib/authenticator.js:289:5)
   at IncomingMessage.req.login.req.logIn (/Users/rolodato/git/saml-sp-linking-sample/node_modules/passport-saml/node_modules/passport/lib/http/request.js:48:29)
   at Strategy.strategy.success (/Users/rolodato/git/saml-sp-linking-sample/node_modules/passport/lib/middleware/authenticate.js:228:13)
   at verified (/Users/rolodato/git/saml-sp-linking-sample/node_modules/passport-saml/lib/passport-saml/strategy.js:54:14)
   at Strategy._verify (/Users/rolodato/git/saml-sp-linking-sample/index.js:26:9)
```

Downgrading to passport 0.2.2 fixed the issue. This was mentioned here: https://github.com/jaredhanson/passport/issues/400

**Comment by jaredhanson** [comment]
Strategies should now depend on `passport-strategy` rather than `passport` itself.

Here's a commit that shows the changes that need to be made:
https://github.com/jaredhanson/passport-http-bearer/commit/886bafa43f6a2e53d555bb28986478bf5f764c2f

**Comment by gnawhleinad** [comment]
The change was made bergie/passport-saml@010f2d4373719cf34b6dc241d51e76d8e4b1c2f2, but it has not been released yet.

**Comment by ploer** [comment]
Should be published as v0.13.0.  Sorry for the delay.

**Comment by SamD** [comment]
This happens again in 0.15, I went back to 0.13 and it works as expected
